identities/tasks.py

```diff
@@ -21,3 +24,23 @@ def run(self, anum, **kwargs):
         return int(anum)+1
 
 the_incr = The_Incr()
+
+
+class DeliverHook(Task):
+    def run(self, target, payload, instance=None, hook=None, **kwargs):
+        """
+        target:     the url to receive the payload.
+        payload:    a python primitive data structure
+        instance:   a possibly null "trigger" instance
+        hook:       the defining Hook object
+        """
+        requests.post(
+            url=target,
+            data=json.dumps(payload),
+            headers={
+                'Content-Type': 'application/json',
+                'Authorization': 'Token %s' % settings.HOOK_AUTH_TOKEN
```

Comment:
Won't the `HOOK_AUTH_TOKEN` need to be per-service that uses the Identity Store and not global? I.e. each application should specify an auth token that allows the Identity Store to prove that it is the thing making the request?